### PR TITLE
Clipbyshape PR

### PR DESCRIPTION
**Pull request by Chris-Schnaufer**
Initial cut of extractor

### CODE REVIEW

**Comment by KristinaRiemer** [comment]
@Chris-Schnaufer how should I go about seeing if this works? Just build and run the dockerfile?

**Comment by Chris-Schnaufer** [comment]
Add context to this request

File: extractors-clipbyshape/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+The clipped files are placed into a new dataset and organized by plot.
+Image pixel size is consistency maintained for all clipped images on a per-plot basis.
+
+A date value is used to organize the clipped files.
+By deffault the local machine date for when the clipping was started is used.
+To force the date to a certain value, include it in the shapefile name with a leading ' - ' separator.
+A trailing ' - ' separator is needed if the date is not the last part of the filename before the file extension.
+The date must use a four digit year format and either come first or last.
+The day and month need to be two digits each.
+The placement of month and day in relation to each other is ignored: DD-MM-YYYY is as good as MM-DD-YYYY and YYYY-DD-MM.
```
Comment (max-zilla) [inline extractors-clipbyshape/README.md]: If DD-MM-YYYY is as good as MM-DD-YYYY how would you differentiate 05-10-2018 being May 10th vs. October 5th?

Comment (robkooper) [reply extractors-clipbyshape/README.md]: Can we please ISO-8601 standard YYYY-MM-DD

Comment (Chris-Schnaufer) [reply extractors-clipbyshape/README.md]: The documentation has been updated

**Pull request merged**